### PR TITLE
Implement update a calendar event func

### PR DESCRIPTION
**Pull request by wiseaidev**
## PR Acceptance Criteria
Related to #47  

## Guidelines
* [X] PR follows the [Contributing guidelines](https://github.com/nylas-samples/nylas-hacktoberfest-2023/blob/main/CONTRIBUTION.md)?
* [X] PR Adheres to our [Code of Conduct](https://github.com/nylas-samples/nylas-hacktoberfest-2023/blob/main/CODE_OF_CONDUCT.md)?
* [X] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Code complete
* [X] A `README.md` available describing how to setup and test the code?
* [X] The code sample in the correct location (i.e. `calendar-event/python/update_calendar_event`)?
* [x] Link added to the repository readme (view `# Code Snippets created by the Nylas Community` with examples) describing what the code sample does for others to try out
* [x] The code can be run using Github Codespaces, so the maintainer can try out the code sample remotely to ensure everything is working before we accept the PR.

## For PR Reviewer use
* [x] Purpose of code sample is clear
* [x] The code sample is readable
* [x] Steps try out the code are provided
* [x] Code sample runs on codespaces environment
* [x] Code works as expected
* [x] PR labels added

### CODE REVIEW

**Comment by atejada** [comment]
Hello @wiseaidev I send you an email with the details on how to finish setting up your V3 account. Once you do that, please update your source code so that it can be tested and accepted.

**Comment by wiseaidev** [comment]
Done!
P.S. Found a bug; The event is not updated!

File: code-samples/calendar/python/update_calendar_event/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    Event ID: 20240503_au947ei7j7t16shff28g32f4tc, Title: Orthodox Good Friday, Date: 2024-05-03
+    Event ID: 20240505_jltlnsbrec2h1mtbtsa49a6ss0, Title: Orthodox Easter Day, Date: 2024-05-05
+    Error update event: Not Found
+
+    Event Update: False
+
+
+    Getting events after event update for Calendar 'Holidays in Lebanon':
+
+    Event ID: 20231029_vabqu54s35oscpe0c4vrpa1ge8, Title: Daylight Saving Time ends, Date: 2023-10-29
```
Comment (wiseaidev) [inline code-samples/calendar/python/update_calendar_event/README.md:112]: This event should be updated accordingly. But there is a bug `Error update event: Not Found`

Comment (wiseaidev) [reply code-samples/calendar/python/update_calendar_event/README.md]: Sorry, a bug from my end.

**Pull request merged**